__tests__/components/HTMLBlock.test.jsx

```diff
@@ -2,6 +2,7 @@ const { render, screen } = require('@testing-library/react');
 const React = require('react');
 const { renderToString } = require('react-dom/server');
 
+// eslint-disable-next-line import/order
```

Comment:
These imports need to be loaded in this order?

Reply:
Let me double check that, but I think eslint was getting confused.

Reply:
Ok, converted to esm and cleaned up to remove the exception.